### PR TITLE
Revert "#5 verify upcoming event"

### PR DESCRIPTION
**Pull request by mehalyna**
Reverts mehalyna/Wr-52.TAQC-Python-TestProject#13

### CODE REVIEW

File: src/pages/common/BaseWrapper.py [hunk 1]
```diff
@@ -87,8 +86,3 @@ def go_to_site(self):
             return site
         except TimeoutError:
             print("Site downloading failed, timeout")
-
-
-    def scroll_down(self, locator, timeout=10):
```
Comment (servettonga) [inline src/pages/common/BaseWrapper.py:92]: I didn't see any usage of this method. Is it necessary to make some elements to appear/load by scrolling?

File: src/pages/LandingPage.py [hunk 2]
```diff
@@ -23,12 +20,3 @@ def __init__(self, driver):
...
         self.join_eventsexpress_btn = ButtonElement(self.JOIN_EVENTSEXPRESS_BTN_CSS, driver)
         self.log_out_btn = ButtonElement(self.LOG_OUT_BTN_CSS, driver)
-
-    def scroll_down_page(self):
-        return self.scroll_down(self.All_PAGE_CSS)
-
-    def get_upcoming_public_event(self):
-        self.scroll_down_page()
-        return self.find_element_by_css(self.UPCOMING_PUBLIC_EVENT)
-
```
Comment (servettonga) [inline src/pages/LandingPage.py:33]: Leave only one empty line between methods and at the bottom of files.

**Review by servettonga — COMMENTED** [review]
Besides styling, I didn't see a problem. Tests pass.

File: src/pages/LandingPage.py (hunk 2 above)
Comment (vtelt) [reply src/pages/LandingPage.py:33]: Resolved in the new commit

**Pull request closed without merging**